fitnessllm_dataplatform/stream/strava/services/api_interface.py

```
@@ -74,10 +87,20 @@
     def set_strava_client(self, access_token: str) -> None:
         """Instantiate strava client."""
         if not access_token:
+            structured_logger.warning(
+                message="No strava access token provided",
+                uid=self.uid,
+                data_source=self.data_source.value.lower(),
+            )
             logger.warning("Strava access token not found in redis")
             return None
```

Ruff (0.8.2)

Do not explicitly `return None` in function if it is the only possible return value

Remove explicit `return None`

(RET501)